scripts/deploy.sh

```diff
@@ -27,8 +27,8 @@ if [[ -z "${API_KEY}" ]]; then
     exit 1
 fi
 
-# Default values for NODE and IPFS
-NODE="${NODE:-https://api.thegraph.com/deploy/}"
+# Default values for GRAPH_NODE and IPFS
+GRAPH_NODE="${GRAPH_NODE:-https://api.thegraph.com/deploy/}"
```

Comment:
`NODE` was conflicting with an env var related to NodeJS so I've updated the name